core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -30,14 +30,24 @@ oppia.directive('simpleEditorSidebar', [function() {
            */
           Ps.initialize(document.querySelector('simple-editor-sidebar'));
           $scope.SUBFIELD_LABELS = [
-            'Multiple choice', 'Correct answer', 'Hints', 'Bridge text'];
+            'Correct answer', 'Hints', 'Bridge text'];
           $scope.questionList = SimpleEditorManagerService.getQuestionList();
           $scope.ID_PREFIX = QuestionIdService.SIDEBAR_PREFIX;
           $scope.getSidebarItemId = function(question, subfieldLabel) {
             return QuestionIdService.getSidebarItemId(
               question.getId(), subfieldLabel
             );
           };
+          $scope.questionType = function(question) {
+            if(question._interactionId==="ItemSelectionInput"){
```

Comment:
Here and elsewhere: add single space before '(' and after ')' and around ===.
Use single quotes (or, in this case, the named constants).